wolfcrypt/src/asn.c

```diff
@@ -6591,27 +6635,42 @@ int wc_GetKeyOID(byte* key, word32 keySz, const byte** curveOID, word32* oidSz,
  * PKCS #5: RFC 8018, A.4 - PBES2-params without outer SEQUENCE
  *                    A.2 - PBKDF2-params
  *                    B.2 - Encryption schemes
+ *                    C   - AlgorithmIdentifier
  */
 static const ASNItem pbes2ParamsASN[] = {
-/*  0 */    { 0, ASN_SEQUENCE, 1, 1, 0 },
-                /* PBKDF2 */
-/*  1 */        { 1, ASN_OBJECT_ID, 0, 0, 0 },
-/*  2 */        { 1, ASN_SEQUENCE, 1, 1, 0 },
-                    /* Salt */
-/*  3 */            { 2, ASN_OCTET_STRING, 0, 0, 0 },
-                    /* Iteration count */
-/*  4 */            { 2, ASN_INTEGER, 0, 0, 0 },
-                    /* Key length */
-/*  5 */            { 2, ASN_INTEGER, 0, 0, 1 },
-                    /* PRF - default is HMAC-SHA1 */
-/*  6 */            { 2, ASN_SEQUENCE, 1, 1, 1 },
-/*  7 */                { 3, ASN_OBJECT_ID, 0, 0, 0 },
-/*  8 */                { 3, ASN_TAG_NULL, 0, 0, 1 },
-/*  9 */    { 0, ASN_SEQUENCE, 1, 1, 0 },
-               /* Encryption algorithm */
-/* 10 */       { 1, ASN_OBJECT_ID, 0, 0, 0 },
-               /* IV for CBC */
-/* 11 */       { 1, ASN_OCTET_STRING, 0, 0, 0 },
+/* PBES2PARAMSASN_IDX_KDF_SEQ                */ { 0, ASN_SEQUENCE, 1, 1, 0 },
```

Comment:
These are long names and make really long lines.
Shorten the names to the unique part for all templates.

Reply:
Shortened to unique parts. Good suggestion!